apps/e2e/integration/supplier_order.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { test, expect } from "@playwright/test";
+
+import { baseURL } from "./constants";
+import { getDbHandle } from "@/helpers";
+import { addBooksToCustomer, createSupplierOrder } from "@/helpers/cr-sqlite";
+import { testOrders } from "@/helpers/fixtures";
+
+test.beforeEach(async ({ page }) => {
+	await page.goto(`${baseURL}orders/suppliers/`);
+});
+
+testOrders("should create a new supplier order", async ({ page, supplier, books }) => {
+	const dbHandle = await getDbHandle(page);
+	await dbHandle.evaluate(addBooksToCustomer, { customerId: 1, bookIsbns: [books[0].isbn] });
+	await dbHandle.evaluate(addBooksToCustomer, { customerId: 1, bookIsbns: [books[0].isbn, books[1].isbn] });
+	await page.goto(`${baseURL}orders/suppliers/`);
+	page.getByRole("button", { name: "Unordered" });
+
+	await page.getByRole("button", { name: "Place Order" }).first().click();
+
```

Comment:
- Add more books
- check the total book count and total value; 
- check the place order button is not shown until at least one row is selected
- check that the selected books and select amount change as expected when: the checkbox is checked; when the "Select Y/Z" is clicked 
- only order a batch of them then check that there is a still an unordered batch remaining